core/src/main/java/brooklyn/util/crypto/FluentKeySigner.java

```diff
@@ -42,7 +44,9 @@
  * X509v3CertificateBuilder drags in an add'l dependency (bcmail) and is harder to use. */
 @SuppressWarnings("deprecation")
 public class FluentKeySigner {
-    
+
+    static { Security.addProvider(new BouncyCastleProvider()); }
```

Comment:
I share the concern but I don't like the idea of mandating a specific entry point as it would mean we can't easily run test classes on their own.

Maybe the thing to do is have a `BrooklynInitialization` class with statics `initBouncyCastle()` and `initAll()` which we can invoke as and where needed?  Start-sequence-wise it wouldn't be any difference but at least the startup logic would be in one place for reference.